#! /bin/sh /usr/share/dpatch/dpatch-run
## gps.h-fix.dpatch by Bernd Zeimetz <bzed@debian.org>
##
## DP: removing autotools guards - inttypes.h should work fine here. #448837

@DPATCH@
diff -urNad gpsd~/gps.h gpsd/gps.h
--- gpsd~/gps.h	2006-12-15 00:43:39.000000000 +0100
+++ gpsd/gps.h	2007-12-05 21:29:14.000000000 +0100
@@ -11,13 +11,7 @@
 #include <sys/types.h>
 #include <sys/time.h>
 #include <stdbool.h>
-#ifdef HAVE_STDINT_H
-#include <stdint.h>
-#else
-#ifdef HAVE_INTTYPES_H
 #include <inttypes.h>
-#endif
-#endif
 #include <limits.h>
 #include <time.h>
 #include <signal.h>
